loader/loader.go

```diff
@@ -620,6 +627,7 @@ func (l *Loader) Close() {
 
 	l.stopLoad()
 
+	l.wg.Wait()
```

Comment:
Do we need to put this `l.wg.Wait()` into `l.stopLoad()` just as we did for `l.workerWg.Wait()`?

Reply:
addressed in https://github.com/pingcap/dm/pull/578/commits/02afc7ebfad754598a6266bdcc1acd1056f6c3a3